src/main/java/seedu/parser/EventParser.java

```diff
@@ -128,19 +127,20 @@ private void append(String mostRecent, String token) throws PacException {
             venue += venue.isEmpty() ? token : (" " + token);
             break;
         default:
-            throw new PacException("EventParser: Invalid flag to append to");
+            throw new PacException("I am not sure '" + token + "' belongs to which flag. "
+                    + "Is a flag provided beforehand?");
```

Comment:
Question in exception makes it confusing...maybe can add a statement that it is an invalid flag.

Reply:
I wanted to suggest to user what to do, instead of only telling them what's wrong.

Anyway, my testing so far shows that this message won't be seen by a user. Either the part without flag is ignored, or the user will see `flag is too short`.
![image](https://user-images.githubusercontent.com/42570474/78350210-63bd7300-75d7-11ea-8dc8-4fde5e533117.png)